tools/test-js-with-node

```diff
@@ -62,6 +62,7 @@ EXEMPT_FILES = make_set(
         "web/src/blueslip.ts",
         "web/src/blueslip_stacktrace.ts",
         "web/src/bootstrap_typeahead.ts",
+        "web/src/branch_pill.ts",
```

Comment:
I think we might want a better name for this. Maybe `integration_branch_pill.ts`? I think we don't need to rename anything else.